src/api/taxonomy/index.ts

```diff
@@ -124,6 +125,16 @@ export const synchronizeTaxonomy = () =>
     }
   });
 
+export const getScheduleLastUpdatedTime = (scheduleId: ScheduleEnum) =>
+  useQuery({
+    queryKey: ['scheduleLastUpdatedTime'],
```

Comment:
Mh, you could use `scheduleId` in queryKey to be aware to avoid an un-wanted cached response.